resources/views/training/apply.blade.php

```diff
@@ -115,8 +115,8 @@
                             </div>
 
                             <div class="form-group form-check">
-                                <input type="checkbox" class="form-check-input" id="exampleCheck1">
-                                <label class="form-check-label" name="englishOnly" for="exampleCheck1">I'm <u>only</u> able to receive training in English</label>
+                                <input type="checkbox" class="form-check-input" id="englishOnly" name="englishOnly" value="true">
```

Comment:
Personally, I don't think we should pre-select this as true.

Reply:
That doesn't pre-select, it just sends in the value=true if it's checked @gustavkauman

Reply:
Ignore my ignorance then 😛